action.yml

```diff
@@ -32,6 +32,10 @@ inputs:
     description: 'path to a .bandit file that supplies command line arguments'
```

Comment:
@coderabbitai, We should clarify the difference between ini and conf files and clarify if they can both be used or are mutually exclusive.

Reply:
@coderabbitai, please open a follow-up GH Issue called "Overhaul Documentation" and include your detailed insights in the new issue's description along with next-steps for updating the inline comments and README file.